classroom.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'student'
+
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classrooms(self)
```

Comment:
- According to the project requirements, you have not properly assigned the classroom to the student, please fix this 🙏 
it should be `student.classrom = self`.